src/app/components/charts/BarChart.tsx

```diff
@@ -11,29 +11,58 @@ import { TooltipContent, type Formatters } from './Tooltip'
 import { COLORS } from '../../../styles/theme/colors'
 
 interface BarChartProps<T extends object> extends Formatters {
+  cartesianGrid?: boolean
   data: T[]
   dataKey: Extract<keyof T, string>
+  rounded?: boolean
+  withBarBackground?: boolean
+  withLabels?: boolean
 }
 
-const BarChartCmp = <T extends object>({ data, dataKey, formatters }: BarChartProps<T>) => (
-  <ResponsiveContainer width="100%" aspect={4}>
-    <RechartsBarChart data={data} margin={{ right: 0, bottom: 0 }}>
-      <CartesianGrid vertical={false} stroke={COLORS.antiFlashWhite3} />
-      <YAxis
-        tick={{ fill: COLORS.brandDark, strokeWidth: 0, fontWeight: 600, fontStyle: 'italic' }}
-        axisLine={false}
-        tickLine={false}
-        type="number"
-        padding={{ bottom: 20 }}
-        tickMargin={0}
-      />
+const barSize = 12
+
+const BarChartCmp = <T extends object>({
+  cartesianGrid,
+  data,
+  dataKey,
+  formatters,
+  rounded,
+  withLabels,
+  withBarBackground,
+}: BarChartProps<T>) => (
+  <ResponsiveContainer width="100%" height={80}>
```

Comment:
Hardcoding the height here messes up `Transactions per day` chart

Reply:
reverted as this is handled in one of your PR